src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -3,21 +3,57 @@
 import static java.util.Objects.requireNonNull;
 
 import seedu.address.model.AddressBook;
+import seedu.address.model.EventBook;
 import seedu.address.model.Model;
 
 /**
- * Clears the address book.
+ * Clears the contact book or events book.
  */
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
+    public static final String MESSAGE_PERSONS_SUCCESS = "Contact book has been cleared!";
+    public static final String MESSAGE_EVENTS_SUCCESS = "Event book has been cleared!";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Clears the address book or event book. "
+            + "Parameters: -p (for person book) or -e (for event book)\n"
+            + "Example: " + COMMAND_WORD + " -p";
+
+    private final boolean isPersonBook;
+
+    public ClearCommand(boolean isPersonBook) {
+        this.isPersonBook = isPersonBook;
```

Comment:
Maybe we can call this 
`isClearPersonBook` instead? `isPersonBook` is a bit awkward